internal/gusserver/gusserver.go

```diff
@@ -109,13 +110,14 @@ func (s *server) heartbeat(w http.ResponseWriter, r *http.Request) error {
 	return nil
 }
 
-func newServer(databaseDir string) (*server, *http.ServeMux, error) {
-	db, err := sql.Open("sqlite", databaseDir+"?mode=rwc")
+func newServer(databaseType, databaseSource string) (*server, *http.ServeMux, error) {
+	fmt.Println(databaseType, databaseSource)
```

Comment:
Remove debugging left-over (or turn into a proper log.Printf() message)

Reply:
👍  turned it into a `log` line